### ISSUE TITLE
Add support for semver

### ISSUE THREAD

**Issue by pombredanne** [royw/Versio#3]
Having an explicit support for a semver http://semver.org/ scheme would be great!

**Comment by royw** [comment]
Agreed.  If no one beats me to it, I'll try to add this around the holidays.

**Comment by jstelzer** [comment]
Just stumbled across this library today. FWIW semver is understood by distutils. Might be nice to mix in/share.

from distutils.version import LooseVersion, StrictVersion
